cmd/kubeadm/app/node/csr.go

```diff
@@ -55,3 +62,71 @@ func PerformTLSBootstrap(connection *ConnectionDetails) (*clientcmdapi.Config, e
 
 	return finalConfig, nil
 }
+
+func RequestNodeCertificate(client certclient.CertificateSigningRequestInterface, privateKeyData []byte, nodeName types.NodeName) (certData []byte, err error) {
```

Comment:
This is the function from the kubelet `csr` sub-package that I had to copy over to make it work with the `api/v1` client from `k8s.io/client-go`. Can you think of a better option right now?